environment.yml

```diff
@@ -7,14 +7,14 @@ channels:
   - gurobi
 dependencies:
   - python=3.7
-#  - sox=14.4.2=h27f72bc_1013
   - gurobi=9.1.0
   - tensorboard
   - pip=20.0.2
   - pip:
     - torch_optimizer
     - torch
-#    - torch_baidu_ctc==0.3.0
+    - tensorflow
+    - torch_baidu_ctc==0.3.0
```

Comment:
This one can be removed? IIRC, it is for common-voice only.

Reply:
wait.. just see the comment that you don't want dependencies for common-voice

Reply:
then why not do the same thing for `sppech`? only it use numba and librosa

Reply:
Okay. Then let's keep it.